test/bridge-test.js

```diff
@@ -14,7 +14,7 @@ describe('Bridge', function() {
     const MIN_REQUIRED_REPORTS = 1;
 
     const commissionAmount = ethers.utils.parseEther(`${COMMISSION_AMOUNT}`);
-    const xTransferAmount = ethers.utils.parseEther(`${XTRANSFER_AMOUNT}`);
+    const xTransferAmount = ethers.utils.parseEther(`${XTRANSFER_AMOUNT}.0001`);
```

Comment:
This is probably not the cleanest way to do it, since it invalidates an externally defined constant, but we do need to see that four decimals work.